host_applications/linux/apps/raspicam/imx219_modes.h

```diff
@@ -74,11 +74,11 @@ struct sensor_regs imx219_8MPix[] =
       {0x0170, 0x01},      // X_ODD_INC [2:0]
       {0x0171, 0x01},      // Y_ODD_INC [2:0]
       {0x0172, 0x03},
-      {0x0174, 0x00},      // Binning Mode H_A
-      {0x0175, 0x00},      // Binning Mode V_A
+      {0x0174, 0x01},      // Binning Mode H_A
+      {0x0175, 0x01},      // Binning Mode V_A
       {0x018C, 0x0A},      // CSI Data Format [15:8]
       {0x018D, 0x0A},      // CSI Data Format [7:0]
-      {0x0301, 0x05},      // VTPXCK_DIV
+      {0x0301, 0x05},      // VTPXCK_DIV // has to be 5
```

Comment:
No need for this additional comment. It was 5 and still is 5.
Anybody messing with the register settings needs to be working from the datasheet.